crates/ark/tests/data_explorer.rs

```diff
@@ -423,8 +421,11 @@ fn test_matrix_support() {
 
         // Get the schema for the test data set.
         let req = DataExplorerBackendRequest::GetSchema(GetSchemaParams {
-            num_columns: 61,
-            start_index: 0,
+            column_indices: vec![
```

Comment:
This could be something like `(0..61).collect_vec()`

Reply:
Indeed. I will do a PR for this.